bin/kafka-run-class.sh

```diff
@@ -30,66 +30,44 @@ if [ -z "$SCALA_BINARY_VERSION" ]; then
 	SCALA_BINARY_VERSION=2.10
 fi
 
-# run ./gradlew copyDependantLibs to get all dependant jars in a local dir
-shopt -s nullglob
-for dir in $base_dir/core/build/dependant-libs-${SCALA_VERSION}*;
-do
-  CLASSPATH=$CLASSPATH:$dir/*
-done
-
-for file in $base_dir/examples/build/libs//kafka-examples*.jar;
-do
-  CLASSPATH=$CLASSPATH:$file
-done
-
-for file in $base_dir/contrib/hadoop-consumer/build/libs//kafka-hadoop-consumer*.jar;
-do
-  CLASSPATH=$CLASSPATH:$file
-done
-
-for file in $base_dir/contrib/hadoop-producer/build/libs//kafka-hadoop-producer*.jar;
-do
-  CLASSPATH=$CLASSPATH:$file
-done
-
-for file in $base_dir/clients/build/libs/kafka-clients*.jar;
-do
-  CLASSPATH=$CLASSPATH:$file
-done
-
-for file in $base_dir/streams/build/libs/kafka-streams*.jar;
-do
-  CLASSPATH=$CLASSPATH:$file
-done
+if [ -z "$PROJECT_NAMES" ]; then
+  PROJECT_NAMES="core tools clients connect stream log4j-appender"
+fi
+PROJECT_NAMES=($PROJECT_NAMES)
+JARPATH=""
+for project in ${PROJECT_NAMES[@]};
+  do
+    case $project in
+      "core")
+      JARPATH=$JARPATH"$base_dir/core/build/dependant-libs-${SCALA_VERSION}/*.jar $base_dir/core/build/libs/kafka_${SCALA_BINARY_VERSION}*.jar "
+      ;;
+      "tools")
+      JARPATH=$JARPATH"$base_dir/tools/build/libs/kafka-tools*.jar $base_dir/tools/build/dependant-libs-${SCALA_VERSION}/*.jar "
```

Comment:
@ijuma Clients doesn't use it. Everything just happens to work because we get them from core.